dispatches/models/fossil_case/concrete_tube/heat_exchanger_tube.py

```diff
@@ -0,0 +1,503 @@
+- HeatExchangerFlowPattern.cocurrent: shell and tube flows from 0 to 1
+(default)
+- HeatExchangerFlowPattern.countercurrent: shell side flows from 0 to 1
+tube side flows from 1 to 0""",
+        ),
+    )
+
+    def build(self):
+        """
+        Begin building model (pre-DAE transformation).
+
+        Args:
+            None
+
+        Returns:
+            None
+        """
+        # Call UnitModel.build to setup dynamics
+        super(HeatExchanger1DDataNoshell, self).build()
+
+        # Set flow directions for the control volume blocks and specify
+        # dicretisation if not specified.
+        if self.config.flow_type == HeatExchangerFlowPattern.cocurrent:
+
+            set_direction_tube = FlowDirection.forward
+
+            if self.config.tube_side.transformation_method is useDefault:
+                _log.warning(
+                    "Discretization method was "
+                    "not specified for the tube side of the "
+                    "co-current heat exchanger. "
+                    "Defaulting to finite "
+                    "difference method on the tube side."
+                )
+                self.config.tube_side.transformation_method = \
+                    "dae.finite_difference"
+
+            if self.config.tube_side.transformation_scheme is useDefault:
+                _log.warning(
+                    "Discretization scheme was "
+                    "not specified for the tube side of the "
+                    "co-current heat exchanger. "
+                    "Defaulting to backward finite "
+                    "difference on the tube side."
+                )
+                self.config.tube_side.transformation_scheme = "BACKWARD"
+        elif self.config.flow_type == HeatExchangerFlowPattern.countercurrent:
+            set_direction_tube = FlowDirection.backward
+
+            if self.config.tube_side.transformation_method is useDefault:
+                _log.warning(
+                    "Discretization method was "
+                    "not specified for the tube side of the "
+                    "counter-current heat exchanger. "
+                    "Defaulting to finite "
+                    "difference method on the tube side."
+                )
+                self.config.tube_side.transformation_method = \
+                    "dae.finite_difference"
+
+            if self.config.tube_side.transformation_scheme is useDefault:
+                _log.warning(
+                    "Discretization scheme was "
+                    "not specified for the tube side of the "
+                    "counter-current heat exchanger. "
+                    "Defaulting to forward finite "
+                    "difference on the tube side."
+                )
+                self.config.tube_side.transformation_scheme = "BACKWARD"
+        else:
+            raise ConfigurationError(
+                "{} HeatExchanger1D only supports cocurrent and "
```

Comment:
This needs to be updated, as it refers to HeatExchanger1D.

However, is all of this needed for this simple case?